lib/features/auto-resize/AutoResize.js

```diff
@@ -245,4 +246,45 @@ function boundsChanged(newBounds, oldBounds) {
     newBounds.width !== oldBounds.width ||
     newBounds.height !== oldBounds.height
   );
+}
+
+/**
+ * Get directions of resize as <nwse>.
+ *
+ * @param {Object} oldBounds - Old bounds.
+ * @param {number} oldBounds.x - Old x.
+ * @param {number} oldBounds.y - Old y.
+ * @param {number} oldBounds.width - Old width.
+ * @param {number} oldBounds.height - Old height.
+ * @param {Object} newBounds - New bounds.
+ * @param {number} newBounds.x - New x.
+ * @param {number} newBounds.y - New y.
+ * @param {number} newBounds.width - New width.
+ * @param {number} newBounds.height - New height.
+ *
+ * @returns {string}
+ */
+function getResizeDirections(oldBounds, newBounds) {
+  var directions = '';
+
+  oldBounds = asTRBL(oldBounds);
+  newBounds = asTRBL(newBounds);
+
+  if (oldBounds.top > newBounds.top) {
+    directions = directions.concat('n');
```

Comment:
Can't we adopt our 'TRBL' naming here? How is 'NWSE' different?

Reply:
It's not different, I'll rename it to TRBL. 🍰